conductor/src/test/java/com/bluelinelabs/conductor/util/AttachFakingFrameLayout.java

```diff
@@ -126,4 +130,34 @@ public void onViewRemoved(View child) {
         super.onViewRemoved(child);
     }
 
+    @Override
+    public boolean post(Runnable action) {
+        if (needDelayPost) {
+            delayedPosts.add(action);
+        } else {
+            return super.post(action);
+        }
+
+        return true;
+    }
+
+    public void runDelayedPosts() {
+        for (Runnable runnable : delayedPosts) {
+            runnable.run();
+        }
+
+        clearDelayedPosts();
+    }
+
+    public void clearDelayedPosts() {
```

Comment:
No need to make a separate method to perform only one action. It gives you 0 benefit if you don't mind to put way more logic into this method.

Reply:
Btw I know this is test code, but there is really no excuse to write so-so code in tests.